packages/yoshi-flow-monorepo/src/scripts/build.ts

```diff
@@ -123,14 +123,13 @@ const build: cliCommand = async function(argv, rootConfig, { apps, libs }) {
     const wixMavenStatics = await import('yoshi-common/build/maven-statics');
     const copyDocker = await import('yoshi-common/build/copy-docker');
 
+    // Run petri-specs once in the root of the monorepo
+    await petriSpecs.default();
```

Comment:
Where'd the config-passing go?

```suggestion
    await petriSpecs.default({
      config: app.config.petriSpecsConfig,
    });
```

Reply:
The config is loaded for each `app`. Petri-specs doesn't run for each app but once in the root. So if we want to add it we should probably use `rootConfig.petriSpecsConfig` instead.

I couldn't find any occurrences of petri-specs and this configuration option in our docs, so I assumed that it's not really important. Actually, i'm not sure what it does. So I thought that it would be good to supply this functionality without the config and add it when there is a need.

Reply:
@amitdahan What do you say? Do you know what is it used for?

Reply:
Found these references to this config object:
1. https://github.com/wix-private/fed-infra/blob/master/petri-specs/lib/petri-specs-build.js
1. https://github.com/wix-private/fed-infra/blob/master/ab-translate/src/abTranslate.js

Having said that, I agree about keeping it out of monorepo unless it's actually being used, which seems it's not :)